docs/src/pages/internals/mapping.js

```diff
@@ -93,6 +93,12 @@ export default () =>
           <td align='center'><span role="img" aria-label="not deserializable">🚫</span></td>
           <td><DotnetReference id='T:System.Array' /> isn’t generic, so Chr.Avro can’t determine its item type.</td>
         </tr>
+        <tr valign='top'>
+          <td><DotnetReference id='T:System.Collections.Generic.HashSet{System.Int32}' /></td>
+          <td align='center'><span role='img' aria-label="serializable">✅</span></td>
+          <td align='center'><span role='img' aria-label="deserializable">✅</span></td>
+          <td><DotnetReference id='T:System.Collections.Generic.HashSet{System.Int32}' /> has a constructor with an <DotnetReference id='T:System.Collections.Generic.IEnumerable{System.Int32}' /> parameter.</td>
+        </tr>
```

Comment:
also update the list above this table (either of the following -> any of the following; add a third bullet)